tests/mockBlockchainClient.ts

```diff
@@ -0,0 +1,60 @@
+import { Tx } from 'chronik-client'
+import { mockedBlockchainTransactions } from './mockedObjects'
+
+export default class MockChronikClient {
+  private _checkNetworkIntegrity = false
+
+  public get checkNetworkIntegrity (): boolean {
+    return this._checkNetworkIntegrity
+  }
+
+  public set checkNetworkIntegrity (value) {
+    this._checkNetworkIntegrity = value
+  }
+
+  getAddressTransactions (_: any): Tx[] {
+    return mockedBlockchainTransactions
+  }
+
+  // getAddressUtxos (_: object): GetAddressUnspentOutputsResponse {
```

Comment:
Commented out tests on this branch?

Reply:
Yeah, all blockchain tests have been commented out and they will be refactored in the end with one tests only PR.

Reply:
Font't use multiple `//`, use one simple `/*  */` block and make sure to have some `// WIP` commented in the top of all of these blocks so we don't forget about fixing it

Reply:
Also, you don't have to comment tests, you can add `.skip` to them so they don't run (but put the `// WIP` anyways ofc)

Reply:
I have to comment anyway because I changed almost all return types, so the code doesn't even compile, that being said, changed single line comment to multiple and added WIP.

Reply:
Got it, but this is still missing here and on `tests/integration-tests/api.test.ts`

Reply:
oh indeed, fixed